internal/app/machined/pkg/controllers/kubespan/manager.go

```diff
@@ -413,12 +413,18 @@ func (ctrl *ManagerController) Run(ctx context.Context, r controller.Runtime, lo
 			return fmt.Errorf("error modifying address: %w", err)
 		}
 
+		mtu := cfgSpec.MTU
+		if mtu == 0 {
+			mtu = constants.KubeSpanLinkMTU
+		}
+
 		for _, spec := range []network.RouteSpecSpec{
 			{
 				Family:      nethelpers.FamilyInet4,
 				Destination: netip.Prefix{},
 				Source:      netip.Addr{},
 				Gateway:     netip.Addr{},
+				MTU:         mtu,
```

Comment:
this makes sense, if we route to the KubeSpan link, we set the MTU

Reply:
Unfortunately, it always should be set.

Case: default external interface mtu=1500, etcd (or other local daemons) creates packet size=1500, it routes to the wireguard link, which adds extra bytes...
As the result: we send packet > 1500 out of the external interface.

Reply:
yep, I saw the same, and this is a nice fix!